File: Client/src/app/core/models/ShortUser.ts
import { FontSizeENUM } from 'src/app/shared/enums/FontSizeEnum';
import { LanguagesENUM } from 'src/app/shared/enums/LanguagesENUM';
import { ThemeENUM } from 'src/app/shared/enums/ThemeEnum';
import { Background } from './Background';

export interface ShortUser {
  id: string;
  name: string;
  email: string;
  photoId: string;
  photoPath: string;
  currentBackground: Background;
  languageId: LanguagesENUM;
  themeId: ThemeENUM;
  fontSizeId: FontSizeENUM;
}

File: Backend/WriteAPI/Common/DTO/Users/ShortUser.cs
using System;
using Common.DatabaseModels.Models.Plan;
using Common.DatabaseModels.Models.Systems;
using Common.DTO.Backgrounds;

namespace Common.DTO.Users
{
    public class ShortUser
    {
        public Guid Id { set; get; }
        public string Name { set; get; }
        public string Email { set; get; }
        public Guid PhotoId { set; get; }
        public string PhotoPath { set; get; }
        public BackgroundDTO CurrentBackground { set; get; }
        public LanguageENUM LanguageId { set; get; }
        public ThemeENUM ThemeId { set; get; }
        public FontSizeENUM FontSizeId { set; get; }
        public BillingPlanTypeENUM BillingPlanId { set; get; }
    }
}
